# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/alsa-firmware/hotfix-uint.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- alsa-firmware-1.2.4/hdsploader/tobin.c.vanilla	2024-04-05 16:13:00.937770256 +0200
+++ alsa-firmware-1.2.4/hdsploader/tobin.c	2024-04-05 16:15:01.309774972 +0200
@@ -8,7 +8,7 @@
 #include "multiface_firmware.dat"
 #include "multiface_firmware_rev11.dat"
 
-int write_bin_file(u_int32_t *array, const char *filename)
+int write_bin_file(uint32_t *array, const char *filename)
 {
 	FILE *out;
 	
--- alsa-firmware-1.2.4/hdsploader/multiface_firmware_rev11.dat.vanilla	2024-04-05 16:14:24.447773528 +0200
+++ alsa-firmware-1.2.4/hdsploader/multiface_firmware_rev11.dat	2024-04-05 16:15:09.914775309 +0200
@@ -1,5 +1,5 @@
 /* stored in little-endian */
-static u_int32_t multiface_firmware_rev11[24413] = {
+static uint32_t multiface_firmware_rev11[24413] = {
 0xffffffff, 0x66aa9955, 0x8001000c, 0xe0000000, 0x8006800c, 0xb0000000,
 0x8004800c, 0xb4fc0100, 0x8003000c, 0x00000000, 0x8001000c, 0x90000000,
 0x8004000c, 0x00000000, 0x8001000c, 0x80000000, 0x0002000c, 0x581a000a,
--- alsa-firmware-1.2.4/hdsploader/multiface_firmware.dat.vanilla	2024-04-05 16:15:19.619775689 +0200
+++ alsa-firmware-1.2.4/hdsploader/multiface_firmware.dat	2024-04-05 16:15:22.017775783 +0200
@@ -1,5 +1,5 @@
 /* stored in little-endian */
-static u_int32_t multiface_firmware[24413] = {
+static uint32_t multiface_firmware[24413] = {
 0xffffffff, 0x66aa9955, 0x8001000c, 0xe0000000, 0x8006800c, 0xb0000000,
 0x8004800c, 0xb4fc0100, 0x8003000c, 0x00000000, 0x8001000c, 0x90000000,
 0x8004000c, 0x00000000, 0x8001000c, 0x80000000, 0x0002000c, 0x581a000a,
--- alsa-firmware-1.2.4/hdsploader/digiface_firmware_rev11.dat.vanilla	2024-04-05 16:15:48.017776802 +0200
+++ alsa-firmware-1.2.4/hdsploader/digiface_firmware_rev11.dat	2024-04-05 16:15:50.086776883 +0200
@@ -1,5 +1,5 @@
 /* stored in little-endian */
-static u_int32_t digiface_firmware_rev11[24413] = {
+static uint32_t digiface_firmware_rev11[24413] = {
 0xffffffff, 0x66aa9955, 0x8001000c, 0xe0000000, 0x8006800c, 0xb0000000,
 0x8004800c, 0xb4fc0100, 0x8003000c, 0x00000000, 0x8001000c, 0x90000000,
 0x8004000c, 0x00000000, 0x8001000c, 0x80000000, 0x0002000c, 0x581a000a,
--- alsa-firmware-1.2.4/hdsploader/digiface_firmware.dat.vanilla	2024-04-05 16:15:54.623777061 +0200
+++ alsa-firmware-1.2.4/hdsploader/digiface_firmware.dat	2024-04-05 16:15:57.130777159 +0200
@@ -1,5 +1,5 @@
 /* stored in little-endian */
-static u_int32_t digiface_firmware[24413] = {
+static uint32_t digiface_firmware[24413] = {
 0xffffffff, 0x66aa9955, 0x8001000c, 0xe0000000, 0x8006800c, 0xb0000000,
 0x8004800c, 0xb4fc0100, 0x8003000c, 0x00000000, 0x8001000c, 0x90000000,
 0x8004000c, 0x00000000, 0x8001000c, 0x80000000, 0x0002000c, 0x581a000a,
